docs/env.md

```diff
@@ -0,0 +1,59 @@
+# Version control with .env
+
+**monasca-docker** leverages the [environmental variable handling][2] of
+[docker-compose][1]. The latest release, a.k.a. versions of the images that
+are used for *master* branch is controlled with the help of [.env](../.env)
+file. In overall the file contains all the images' versions and resembles the
```

Comment:
-"In overall"
+"In summary, " / "To summarize, " / "Basically, " / "From a general point of view, " / "To keep things simple, "

Reply:
Done